server/rooms.ts

```diff
@@ -102,6 +102,7 @@ export interface RoomSettings {
 	dataCommandTierDisplay?: 'tiers' | 'doubles tiers' | 'numbers';
 	requestShowEnabled?: boolean | null;
 	showEnabled?: GroupSymbol | true;
+	creationTime?: number | null;
```

Comment:
...this is still not optional. It should also go at the top with the other non-optional properties.